src/cheshire/core.clj

```diff
@@ -8,9 +8,34 @@
                                        JsonGenerator
                                        JsonGenerator$Feature)
            (com.fasterxml.jackson.dataformat.smile SmileFactory)
+           (cheshire.prettyprint CustomPrettyPrinter)
            (java.io StringWriter StringReader BufferedReader BufferedWriter
                     ByteArrayOutputStream OutputStream Reader Writer)))
 
+(def ^:private default-pretty-print-options
+  {:indentation 2
+   :indent-arrays? false
+   :indent-objects? true
+   :before-array-values nil
+   :after-array-values nil
+   :object-field-value-separator nil})
+
+(defn- create-pretty-printer
```

Comment:
I mean if they aren't private, then someone can do:

```clj
(def my-pretty-printer (create-pretty-printer (assoc default-pretty-print-options :indentation 3)))
(generate-string {:foo :bar} {:pretty my-pretty-printer})
```

This is a helper method (and map) for people that want to construct a single instance of their own pretty printer